.chloggen/fix-internal-settings-log-level-for-otel-ta.yaml

```
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

```suggestion
change_type: bug_fix
```
Do you think `bug_fix` would be more accurate?

---

I think it is not a bug per se and users won't ever see this on the TA since when we release the TA logging to splunkd.log the collector will be already fixed.